week_4/quiz-trivia-app/src/app/data/quiz.model.ts

```diff
@@ -1,7 +1,11 @@
-export class Quiz {
-  constructor(
-    public title: string,
-    public icon: string,
-    public questions: { question: string; options: string[]; answer: string }[]
-  ) {}
+export interface Quiz {
+  title: string;
+  icon: string;
+  questions: { question: string; options: string[]; answer: string }[];
```

Comment:
use QuestionData instead of { question: string; options: string[]; answer: string }

Reply:
done